v3io/dataplane/transport/httpclient.py

```diff
@@ -166,6 +168,11 @@ def _send_request_on_connection(self, request, connection):
                     connection=connection,
                 )
                 connection.close()
+                if request.body and hasattr(request.body, "seek") and hasattr(request.body, "tell"):
```

Comment:
Any reason for this condition to be very slightly different from the one on line 158?